### ISSUE TITLE
Cannot get AWSCore to import on iOS

### ISSUE THREAD

**Issue by woodpav** [awslabs/aws-sdk-react-native#60]
I cannot get it to build following the instructions in the guide in #35 .

I get the error `#import <AWSCore/AWSCore.h>` file not found.

<img width="430" alt="screen shot 2017-06-14 at 11 27 06 am" src="https://user-images.githubusercontent.com/22600052/27148243-6c5c1652-50f4-11e7-8368-f01b0ad0a888.png">

I have AWSCore.framework in Libraries > Core.xcodeproj > Frameworks. Expanding it shows that it is empty (not sure if normal). 

<img width="220" alt="screen shot 2017-06-13 at 7 09 17 pm" src="https://user-images.githubusercontent.com/22600052/27148284-8ced2b54-50f4-11e7-9c14-63dfb2a07cfe.png">

I also have a AWSCore.framework listed directly below the Project. Expanding that also shows nothing. 

It looks like the header search paths are working correctly because the file that is throwing the error is included from `node_modules/aws-sdk-react-native-core/ios/Frameworks`. _Project_ build settings frameworks search paths has `$(SRCROOT)/../node_modules/aws-sdk-react-native-core/ios/Frameworks`. 

Opening that folder in finder shows AWSCore.framework. Expanding that shows Modules and Versions. 

Ive also got AWSCore.framework in Target > General > Embedded Libraries. 

Any idea what could be going wrong?

Its worth noting that the Android build works.

**Comment by woodpav** [comment]
Yes! I figured it out!

**TLDR**: Use `npm install` not `yarn add`

Using `yarn add aws-sdk-react-native-core-0.0.2.tgz` breaks `AWSCore.framework`:

<img width="187" alt="screen shot 2017-06-19 at 3 17 51 pm" src="https://user-images.githubusercontent.com/22600052/27308273-7ebb8de8-5502-11e7-8a03-caed47243e44.png">

Using `npm install aws-sdk-react-native-core-0.0.2.tgz` leaves `AWSCore.framework` as it should be:

<img width="190" alt="screen shot 2017-06-19 at 3 16 47 pm" src="https://user-images.githubusercontent.com/22600052/27308274-82032d9e-5502-11e7-8296-4d6fa831a024.png">
